stage2/tasks/code-review/MentorEvaluationCriteria.md

```diff
@@ -0,0 +1,43 @@
+# Mentor Evaluation Criteria for Code Review Task
+
+## Introduction
+
+This document outlines the evaluation criteria for mentors assessing the code review task. The scoring is based on the student's ability to effectively identify issues, provide suggestions, and maintain professional communication.
+
+## Evaluation Criteria
+
+### 1. Identification of Code Issues (Total: 50 points)
+
+- **2 points for each identified issue**.
+- Maximum of 50 points for this criterion.
+- Focus on identifying inefficient code, poor practices, or errors.
```

Comment:
I think we should add a few examples of poor practices, like:
_Focus on identifying inefficient code, poor practices (code repeat, overcomplicated code, useless code, misleading naming, etc.), or errors._